libsplinter/src/biome/rest_api/actix/mod.rs

```diff
@@ -27,6 +27,12 @@ pub(super) mod key_management;
 pub(super) mod login;
 #[cfg(feature = "biome-credentials")]
 pub(super) mod register;
+#[cfg(all(
```

Comment:
It should be fairly rare that things are guarded by multiple features; it implies that some of the items are cross-cutting concerns that can't be stated as dependencies. Here, I don't believe that is the case. This should probably only be guarded by biome-refresh-token with biome-refresh-token dependent on what it needs.